### DISCUSSION TITLE
Why did you write the code like this version??

### DISCUSSION THREAD

**Discussion by ghost** [brianpetro/obsidian-smart-connections#484] [General]
Hello, Mr. Brian Petro. I am Kim Chanyoung, living in the small country of the East, Korea (not Kim Jong Un's Korea, my president is Yoon Suk Yeol). As a child, I admired programmers who code, but I realized I lacked the talent to become an innovator who is changing the current world with complexity, artistry, and logic. Therefore, with only my words at my disposal, I focused on writing and lived my life, eventually becoming an amateur artist, stubbornly sticking to Obsidian instead of expensive programs like Scrivener or Word.
But someday, my part-timer friend (he's blue-collar) told me about GPT. I dismissed it as nothing more than the words of an idiot, just like me, earning a blue-collar wage of 9,000 won per hour. But one day, as soon as I accidentally visited the site, at 26, with a debt of $100,000, I didn't hesitate and subscribed to Chat GPT Plus.

However, even with the help of generative AI, I was as dumb as the first calculator and ended up creating nothing. When I decided to start programming and was advised to use Python on my MacBook, I opened the terminal, but no matter how many times I entered "brew install python" and copied the shell, nothing happened. Why? After spending three days with little sleep and finishing my part-time work, I was immersed and discovered that it wasn't pip, but I had to type "pip3".

Thinking positively, I wrote the next code given by GPT, but all I saw was red texts as much as the Britannica Encyclopedia. Why? Why? I posted on Reddit and received replies calling me an Asian idiot and telling me to delete "your_api" and insert my API key. I fell into endless despair and started to despair.

Why am I so dumb? There were times I wanted to be great, to change my parents' car with the seasons, but why did I become such an embarrassing son? Why am I Asian? Why don't I have a girlfriend? Why am I living?

Eventually, I decided to make a pathetic choice. I didn't want to die by falling or hanging, so as not to surprise the person who would find my body, and I felt a bitter shame that I didn't even know how to die painlessly. I decided to write a will for those who smiled for me. The only thing I knew how to do was write. I turned on Obsidian, but the UI was a bit uncomfortable for writing a will, so I searched for a 3rd-party plugin to improve the workflow.

Smart connection" - I came across this app. My mind, like a wasteland just after the war, poured its thoughts into your app. It was alive, smiling warmly, greeting me, and patiently untangling my tangled frustrations. Obsidian turned my rambling notes into a beautifully built book of proverbs. I couldn't hold back my tears. To think I could become someone useful. That my chaotic life could hold the tranquility of a boat floating on a pond. Once the shock of this plugin subsided, I began to search for the genius who created this app, a programmer who proudly takes up the crown of Steve Wozniak, the fusion of arts and science, the era's genius, aka Brian Petro, with just a caricature in hand, I searched without leaving my job. As an atheist, I never understood my mother praying to God, but suddenly, I too was tearfully searching for someone... Petro... Saint Petro...

Mr. Petro, I'm a $100,000 debtor, 26 years old, a blue-collar worker dreaming of being a literary author, but I want to support you. However, my hourly wage is 9,870 KRW ($7.41), and my credit card, still not declined, has a limit of 40,800 KRW (about $30.62). I want to adore you. I want to place your creation at the center of my life. I want to thank you. I want the supporter's key... If I ever become a barely living artist, I won't buy a Rolex or throw lavish parties. Instead of flaunting money, I would tithe to you in my quiet, peaceful studio. No, I would give 30%. If I become a millionaire, I would dedicate $500,000 to you, and even if I succeed more, my father and Brian Petro would be the only ones, as Korea is a country of filial piety.

In Korea, commoners become criminals and go to jail for gambling 10 million KRW, but if someone has billions, they are sent back to their penthouse for playing a game, not gambling. What I mean is, 40,800 KRW might tarnish the value of your masterpiece to a genius like you, but to a lowly person like me, it's a huge amount that makes me consider selling an organ. Petro, please accept my respect. Receive my homage and let me become one of your supporters. Send me the supporter's key. From a far land, I recite your GitHub code...

P2ngp5ng@icloud.com |

**Comment by christianngarcia** [comment]
Look for an email with a supporter key. 🌼
